src/game/boe.specials.cpp

```diff
@@ -1917,7 +1917,11 @@ void queue_special(eSpecCtx mode, eSpecCtxType which_type, spec_num_t spec, loca
 	queued_special.type = which_type;
 	queued_special.mode = mode;
 	queued_special.trigger_time = univ.party.age;
-	special_queue.push(queued_special);
+	// FIXME: I forced calling the leave special just to avoid calling them outside, ie. with town_num=200
```

Comment:
You call this a hack and leave a FIXME. Did you find a better solution later on?

Reply:
I keep this modification, it works. But I still think it's a hack (not an example of good programming).